package.json

```diff
@@ -12,42 +12,39 @@
     "logging",
     "winston"
   ],
+  "exports": {
+    ".": {
+      "import": "./lib/index.mjs",
+      "default": "./lib/index.js"
+    }
+  },
   "scripts": {
+    "build": "yarn run lint && yarn run test -- --run && bunchee",
     "lint": "eslint src",
-    "test": "jest",
+    "test": "vitest",
     "test:watch": "npm run test -- --watch --onlyChanged",
-    "build": "babel src -d lib --ignore __tests__",
     "prepublish": "npm run lint && npm run test && rimraf ./lib && npm run build"
   },
   "files": [
     "lib"
   ],
   "devDependencies": {
-    "babel-cli": "6.26.0",
-    "babel-eslint": "7.2.3",
     "babel-jest": "21.2.0",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-preset-env": "1.6.1",
-    "eslint": "4.6.1",
-    "eslint-config-molindo": "1.0.0",
-    "eslint-plugin-css-modules": "2.7.5",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-import-order-autofix": "0.8.3",
-    "eslint-plugin-jsx-a11y": "6.0.2",
-    "eslint-plugin-prettier": "2.2.0",
-    "eslint-plugin-react": "7.3.0",
-    "jest": "21.2.1",
-    "prettier": "1.7.0",
-    "supertest": "3.0.0"
+    "bunchee": "^5.1.2",
+    "eslint": "8.56.0",
+    "eslint-config-molindo": "7.0.0",
+    "prettier": "3.2.5",
+    "supertest": "6.3.4",
+    "vitest": "^1.3.1"
   },
   "dependencies": {
-    "body-parser": "^1.18.2",
-    "express": "^4.16.2",
-    "express-winston": "^2.4.0",
-    "winston": "^2.4.0"
+    "body-parser": "^1.20.2",
+    "express": "^4.19.2",
+    "express-winston": "^4.2.0",
+    "winston": "^3.13.0",
```

Comment:
Can you reference "Closes #16" in the PR description to auto-close https://github.com/molindo/molindo-node-logger/issues/16 on merge?